test/test_rules.py

```diff
@@ -4,92 +4,116 @@
 
 
 class TestRules(unittest.TestCase):
-    def test_if_three_in_horizontal_first_row_return_true(self):
-        rules = Rules()
-        current_board = ["X", "X", "X", "O", "5", "6", "7", "8", "9"]
-        winner = rules.is_winner_horizontal(current_board)
-        self.assertEqual(True, winner)
-
-    def test_if_three_in_horizontal_second_row_return_true(self):
-        rules = Rules()
-        current_board = ["1", "2", "3", "X", "X", "X", "7", "8", "9"]
-        winner = rules.is_winner_horizontal(current_board)
-        self.assertEqual(True, winner)
-
-    def test_if_three_in_horizontal_third_row_return_true(self):
-        rules = Rules()
-        current_board = ["1", "2", "3", "4", "5", "6", "X", "X", "X"]
-        winner = rules.is_winner_horizontal(current_board)
-        self.assertEqual(True, winner)
+    def setUp(self):
+        self.rules = Rules()
+        self.starter_board = ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+
+    def test_horizontal_rows_return_true_when_three_in_a_row(self):
+        test_top_row = ["X", "X", "X", "O", "5", "6", "7", "8", "9"]
```

Comment:
@Aoxian to be clear - is this correct?  The test is doing one thing (I think??) - testing that `is_winner_horizontal` returns true if the board has a horizontal win, and we test it with multiple boards. Or do these tests need to be fixed/changed too?

Reply:
you got it